articles/connections/social/google.md

```diff
@@ -14,6 +14,8 @@ seo_alias: google
 
 To connect your Auth0 client to Google and Google Apps, you will need to generate a *Client ID* and *Client Secret* in a Google project, copy these keys into your Auth0 settings, and enable the Connection.
 
+If you are looking to manage authentication in your native Android app [see the Android quickstart](/quickstart/native/android).
+
```

Comment:
This looks a bit out of context, maybe we need something more generic.

I think it would look better if instead of this, we added a paragraph at the end (for example, `10. Next steps`) explaining that now that the user has a working connection the next step is to use it in the app to authenticate a user, and add a bunch of links to our quickstarts and more resources. For example:
- [Quickstarts for Mobile / Native Apps](/quickstart/native)
- [Quickstarts for Single Page Apps](/quickstart/spa)
- [Quickstarts for Web Apps](/quickstart/webapp)
- For more information on client authentication refer to [Client Authentication](/client-auth)

Actually I would add this to all our connections docs. You can add this paragraph as a [fragment document](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#reusing-content) and reuse it from all the extensions docs.

What do you think?